### PR TITLE
Broken link fsharp

### LINKED ISSUE #102: Broken link for "The syntax highlighting uses F# Formatting"
The broken link is at the bottom right, "The syntax highlighting uses [F# Formatting](http://tpetricek.github.io/FSharp.Formatting/)"

It may be intended to point to https://fsprojects.github.io/FSharp.Formatting/ now

### PR DESCRIPTION
**Pull request by anthonyflores-github**
Fixes #102 

I changed the link.

Thanks

### CODE REVIEW

File: templates/snippet.html [hunk 1]
```diff
@@ -62,8 +62,6 @@ <h1>{{ model.Details.Title }}</h1>
         <span class="caret"></span>
       </button>
       <ul class="dropdown-menu">
-        <li><a href="http://tsunami.io/cloud_tsunami.html?initurl=http://fssnip.net/raw/{{ model.Details.Id | format_id }}">tsunami.io</a></li>
-        <li><a href="http://www.tryfsharp.org/Create?codeUrl=http://fssnip.net/raw/{{ model.Details.Id | format_id }}">tryfsharp.org</a></li>
         <li><a href="http://tryfs.net/snippets/snippet-{{ model.Details.Id | format_id }}">tryfs.net</a></li>
```
Comment (dsyme) [inline templates/snippet.html:65]: This is also broken isn't it?

Comment (anthonyflores-github) [reply templates/snippet.html:65]: which link is broken? tryfs.net, it does not return to the right place.
I can remove it too.

Comment (anthonyflores-github) [reply templates/snippet.html:65]: I removed the broken link on the right branch. #104

**Comment by tpetricek** [comment]
Hi @anthonyflores-github! Thanks for all the work on this! I merged your three PRs into a single branch, tested it locally and I've merged this into master (so it should be deployed soon). 

I did some tweaks in your test page - it was based on the edit page, which worked nicely, but I deleted all the code that was no longer needed, so that it just shows the Sphere Engine widget and nothing else.

It would be nice if the widget was a bit bigger - but adding `style="height:800px"` did nothing. If you had time, could you investigate if we can make it bigger somehow?

**Pull request merged**